docs/src/guides/launch.md

```diff
@@ -37,6 +37,89 @@ To completely reset the dev environment:
   zkstack ecosystem init
   ```
 
+### Advanced local setup with ZKsync Gateway
+
+- Stop services from any other previous setup:
```

Comment:
```suggestion
- Delete services from any other previous setup:
```

I think stop may make somebody mistakengly believe this only stops stuff. AFAIK this deletes volumes

Reply:
fixed in last commit